README.md

```diff
@@ -42,6 +42,49 @@ production control planes.
 
 ![UXP in Upbound Cloud](docs/media/uxp-in-ubc.png)
 
+### Installation With Helm 3
+
+Helm requires the use of `--devel` flag for versions with suffixes, like
+`v1.2.1-up.3`. But Helm repository we use is the stable repository so use of that
+flag is only a workaround, you will always get the latest stable version of UXP.
+
+1. Create the namespace to install UXP.
+
+   ```console
+   kubectl create namespace upbound-system
+   ```
+
+1. Add `upbound-stable` chart repository.
+
+   ```console
+   helm repo add upbound-stable https://charts.upbound.io/stable && helm repo update
+   ```
+
+1. Install the latest stable version of UXP.
+
+   ```console
+   helm install uxp --namespace upbound-system upbound-stable/universal-crossplane --devel
+   ```
+
+### Upgrade from upstream Crossplane
+
+#### Using up CLI
+
+1. Upgrade the Crossplane installation in given namespace.
```

Comment:
nitpick: this sounds a little bit like you are just upgrading to a new Crossplane version. I wonder if we could just remove the text from these sections given that there is only one step?

Reply:
Removed and added some details about upgrade and version match.